meta-lmp-base/recipes-support/docker-image-preload/docker-image-preload/docker-image-preload.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker container images preloading
+
+After=local-fs.target
+Before=docker.service
+
+ConditionPathExists=/var/sota/import/installed_versions
+ConditionPathExistsGlob=/var/sota/import/*.tar
+
+[Service]
+Type=oneshot
```

Comment:
I believe it is also recommended to use 'RemainAfterExit=yes' here, otherwise the service will show up as dead after being executed. This is not particularly clear to, but it seems the standard.

Reply:
I read about it and tried it, but I don't see any reason it's needed in this particular case.

Reply:
I just double-checked it once more, I don't see any difference in the service behavior with or without  RemainAfterExit=yes.

Reply:
Actually, there is some difference. With `RemainAfterExit=yes` and if the conditions are met and the ExecStart* scripts return(s) 0 then the service state will be at `Active: active (exited)` state while without `RemainAfterExit=yes` or if the conditions are not met or one of ExecStart* scripts fail then it will be at `Active: inactive (dead)`. Also, the service will be at this state `Active: inactive (dead)` after the second boot regardless of `RemainAfterExit=yes`.
If the service is at `Active: inactive (dead)` then the docker service restart causes a restart of the given service. It's not quite what we need but it doesn't cause any issue anyway.

Reply:
Cool, thanks for investigating that one.